src/OpalCompiler-Core/CopiedLocalVariable.class.st

```diff
@@ -2,7 +2,7 @@
 A copying variable is an arg or temp var that is copied into a block that later reads this variable
```

Comment:
The comment still uses an old word copying

Reply:
added to backlog